core/domain/exp_domain.py

```diff
@@ -3671,6 +3671,24 @@ def get_all_html_content_strings(self):
 
         return html_list
 
+    def get_all_html_strings(self):
+        """Gets all html strings used in this exploration with the
+        corresponding state name.
+
+        Returns:
+            dict. Mapping html content strings to state name.
+        """
+        html_list = []
+        state_names_to_html_strings_mapping = {}
+        for state_name in self.states:
+            state = self.states[state_name]
+            content_html = state.content.html
+            interaction_html_list = (
+                state.interaction.get_all_html_content_strings())
```

Comment:
This should also be get_all_html_strings()

Reply:
Done.